crates/bevy_pbr/src/meshlet/mod.rs

```diff
@@ -241,6 +192,21 @@ impl Plugin for MeshletPlugin {
                         .in_set(RenderSystems::QueueMeshes)
                         .before(queue_material_meshlet_meshes),
                 ),
+            )
+            .add_systems(
+                Core3d,
+                (
+                    meshlet_visibility_buffer_raster.before(early_shadow_pass),
+                    meshlet_prepass
+                        .after(early_shadow_pass)
+                        .before(Core3dSystems::EndPrepasses),
+                    meshlet_deferred_gbuffer_prepass
+                        .after(meshlet_prepass)
+                        .before(Core3dSystems::EndPrepasses),
+                    meshlet_main_opaque_pass
+                        .after(Core3dSystems::StartMainPass)
+                        .before(Core3dSystems::EndMainPass),
```

Comment:
this is not a faithful transcription of the orderings, not sure why its different or if it matters though